src/ProgressiveColonizationSystem/PksOnStationParameterFactory.cs

```diff
@@ -0,0 +1,107 @@
+﻿using ContractConfigurator;
+using Contracts;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace ProgressiveColonizationSystem
+{
+    /// <summary>
+    ///  Use this if it seems like a good idea to require that the kerbal make it all the way
+    ///  to a base.  Otherwise use the ReachState configuration.
+    /// </summary>
+    public class PksOnStationParameterFactory
+        : ParameterFactory
+    {
+        private string body;
```

Comment:
I would think that in order to populate from the config, all I should have to do here is throw a [kspfield] on it, but I don't see evidence of anything else doing it.

Reply:
I didn't know about KSPField when I first wrote this stuff.  It could be there's something special that I need to do that I'm not doing (like having ParameterFactory inherit from IConfigNode).